optimum/neuron/pipelines/diffusers/pipeline_stable_diffusion.py

```diff
@@ -18,11 +18,16 @@
 from typing import Any, Callable, Dict, List, Optional, Union
 
 import torch
-from diffusers import StableDiffusionPipeline
-from diffusers.loaders import LoraLoaderMixin, TextualInversionLoaderMixin
-from diffusers.pipelines.stable_diffusion import StableDiffusionPipelineOutput
-from diffusers.pipelines.stable_diffusion.pipeline_stable_diffusion import rescale_noise_cfg
-from diffusers.utils import randn_tensor
+
+from ....utils import is_diffusers_available
+
+
+if is_diffusers_available():
+    from diffusers import StableDiffusionPipeline
+    from diffusers.loaders import LoraLoaderMixin, TextualInversionLoaderMixin
+    from diffusers.pipelines.stable_diffusion import StableDiffusionPipelineOutput
+    from diffusers.pipelines.stable_diffusion.pipeline_stable_diffusion import rescale_noise_cfg
+    from diffusers.utils.torch_utils import randn_tensor
```

Comment:
It will fail if diffusers is not installed no?
Because then we try to inherint from a class that was not imported.

Reply:
The change is to tackle the import issue you met when launching training tests. I would raise an import error is diffusers not installed, wdyt?

Reply:
Where would you raise that?